app/modules/intelligence/tools/change_detection/change_detection_tool.py

```
@@ -598,65 +600,65 @@ async def get_code_changes(self, project_id):
                                 default_commit_shas = [c.sha for c in default_commits]
 
                                 if commit.sha in default_commit_shas:
-                                    logging.info(
+                                    logger.info(
                                         f"[CHANGE_DETECTION] Reached common ancestor at commit {commit.sha[:7]}"
                                     )
                                     break
                             except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)